Microsoft.Toolkit.Uwp.Services/Services/OneDrive/OneDriveStorageItem.cs

```diff
@@ -171,6 +171,7 @@ public Item OneDriveItem
         }
 
         /// <summary>
+
```

Comment:
why adding this new empty line ?

Reply:
Well, I don't know. Maybe my mistake hitting the return key.